src/panels/lovelace/components/hui-theme-select-editor.ts

```diff
@@ -0,0 +1,71 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import "@polymer/paper-button/paper-button";
+import { TemplateResult } from "lit-html";
+
+import { HomeAssistant } from "../../../types";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+
+export class HuiThemeSelectionEditor extends hassLocalizeLitMixin(LitElement) {
```

Comment:
Great job on breaking out all the different reusable editor components ! 

One would _almost_ think that we should have gone for MVP and have just 1 editor component per PR 😛

Reply:
Probably. Will keep that in mind when creating new ones